core/storage/feedback/gae_models.py

```diff
@@ -287,13 +423,34 @@ def create(cls, exploration_id, thread_id, author_id, exploration_version,
 
     @classmethod
     def _get_instance_id(cls, exploration_id, thread_id):
+        """Returns the full thread ID corresponding to the
+        given exploration ID and thread ID.
+
+         Args:
+             exploration_id: str. ID of the exploration the thread
+                belongs to.
+             thread_id: str. ID of the thread.
+
+         Returns:
+             str. Returns the full thread ID corresponding to
+                the given exploration ID and thread ID.
+        """
         return '.'.join([exploration_id, thread_id])
 
     @classmethod
     def get_by_exploration_and_thread_id(cls, exploration_id, thread_id):
         """Gets a suggestion by the corresponding exploration and thread id's.
 
-        Returns None if it doesn't match anything."""
+        Args:
+            exploration_id: str. ID of the exploration to which
+                SuggestionModel belongs.
+            thread_id: str. ID of the thread.
```

Comment:
Thread ID of the suggestion thread.

(When I first read this, I wondered: "what thread?")